azure-toolkit-libs/azure-toolkit-containerapps-lib/src/main/java/com/microsoft/azure/toolkit/lib/containerapps/containerapp/Replica.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *
+ */
+
+package com.microsoft.azure.toolkit.lib.containerapps.containerapp;
+
+import com.azure.resourcemanager.appcontainers.models.ReplicaContainer;
+import com.azure.resourcemanager.resources.fluentcore.arm.ResourceId;
+import com.microsoft.azure.toolkit.lib.common.model.AbstractAzResource;
+import com.microsoft.azure.toolkit.lib.common.model.AbstractAzResourceModule;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nonnull;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class Replica extends AbstractAzResource<Replica, Revision, com.azure.resourcemanager.appcontainers.models.Replica> {
+    protected Replica(@Nonnull String name, @Nonnull String resourceGroupName, @Nonnull ReplicaModule module) {
+        super(name, resourceGroupName, module);
+    }
+
+    protected Replica(@Nonnull Replica origin) {
+        super(origin);
+    }
+
+    protected Replica(@Nonnull com.azure.resourcemanager.appcontainers.models.Replica remote, @Nonnull ReplicaModule module) {
+        super(remote.name(), ResourceId.fromString(remote.id()).resourceGroupName(), module);
+    }
+
+    @NotNull
+    @Override
+    public List<AbstractAzResourceModule<?, ?, ?>> getSubModules() {
+        return Collections.emptyList();
+    }
+
+    @NotNull
+    @Override
+    public String loadStatus(@NotNull com.azure.resourcemanager.appcontainers.models.Replica remote) {
+        return Status.UNKNOWN;
```

Comment:
check if have status

Reply:
![Snipaste_2023-04-19_09-40-30](https://user-images.githubusercontent.com/110525714/232944540-6b0a6ad3-6a8b-4802-b042-1ac8ff37c1a0.png)
The provision status shown in the picture is the status of revision instead of replica. It seems no appropriate attribute value to be used as replica status.
![Snipaste_2023-04-19_09-43-50](https://user-images.githubusercontent.com/110525714/232944930-67953537-a42b-4248-8bec-2cf2d3912f85.png)